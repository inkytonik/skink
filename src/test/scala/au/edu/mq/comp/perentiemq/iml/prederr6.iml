// Non-associative operator
def prederr1(a) : (b) {
    assert (a > b > c);
}
