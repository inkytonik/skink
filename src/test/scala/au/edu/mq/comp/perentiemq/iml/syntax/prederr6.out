src/test/scala/au/edu/mq/comp/perentiemq/iml/syntax/prederr6.iml:3:19: ')' expected
    assert (a > b > c);
                  ^
